### PR TITLE
Solidity

### PR DESCRIPTION
**Pull request by pecraveiro**
Icon request: Solidity #864 - https://github.com/devicons/devicon/issues/864#issuecomment-1010219346

# **New icon section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date

Solidity Original SVG: https://upload.wikimedia.org/wikipedia/commons/9/98/Solidity_logo.svg
Solidity Original SVG: https://docs.soliditylang.org/en/v0.8.11/_static/logo.svg

---
# **New feature section**

## Double check these details before you open a PR

- [ ] PR does not match another non-stale PR currently opened
- [ ] PR name matches the format *New Feature: a brief description of feature*

### CODE REVIEW

File: icons/solidity/solidity-original.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 523.9 813"><style type="text/css">.st0{opacity:0.45;enable-background:new;}.st1{opacity:0.6;enable-background:new;}.st2{opacity:0.8;enable-background:new;}</style><path class="st0" d="M391.9 0l-130.7 232.3H0L130.6 0 391.9 0"/><path class="st1" d="M261.2 232.3h261.3L391.9 0h-261.3L261.2 232.3z"/><path class="st2" d="M130.6 464.5l130.6-232.2L130.6 0 0 232.3 130.6 464.5z"/><path class="st0" d="M131.9 813l130.7-232.3h261.3L393.2 813H131.9"/><path class="st1" d="M262.6 580.7h-261.3L131.9 813H393.2L262.6 580.7z"/><path class="st2" d="M393.2 348.5l-130.6 232.2L393.2 813l130.7-232.3L393.2 348.5z"/></svg>
```
Comment (Panquesito7) [inline icons/solidity/solidity-original.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 523.9 813"><style type="text/css">.st0{opacity:0.45;enable-background:new;}.st1{opacity:0.6;enable-background:new;}.st2{opacity:0.8;enable-background:new;}</style><path class="st0" d="M391.9 0l-130.7 232.3H0L130.6 0 391.9 0"/><path class="st1" d="M261.2 232.3h261.3L391.9 0h-261.3L261.2 232.3z"/><path class="st2" d="M130.6 464.5l130.6-232.2L130.6 0 0 232.3 130.6 464.5z"/><path class="st0" d="M131.9 813l130.7-232.3h261.3L393.2 813H131.9"/><path class="st1" d="M262.6 580.7h-261.3L131.9 813H393.2L262.6 580.7z"/><path class="st2" d="M393.2 348.5l-130.6 232.2L393.2 813l130.7-232.3L393.2 348.5z"/></svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! I suggest keeping the `solidity-original.svg` file only. You also need to update `devicon.json`. https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#updateDevicon

**Comment by pecraveiro** [comment]
Updated devicon.json;
Deleted the Solididty folder and added another one;
Resized the solidity-original.svg;

**Comment by Thomas-Boi** [comment]
Hi @pecraveiro ,

It seems like we already have [an icon for solidity](https://github.com/devicons/devicon/tree/develop/icons/solidity) that was merged recently. Unfortunately, we would be unable to accept your PR. Thank you very much for your work and we apologize for the inconvenience.

**Comment by pecraveiro** [comment]
That's ok! Have a nice day.

**Pull request closed without merging**